samples/snippets/create_training_pipeline_image_classification_sample.py

```diff
@@ -30,18 +30,18 @@ def create_training_pipeline_image_classification_sample(
     # Initialize client that will be used to create and send requests.
     # This client only needs to be created once, and can be reused for multiple requests.
     client = aiplatform.gapic.PipelineServiceClient(client_options=client_options)
-    training_task_inputs_dict = {
-        "multiLabel": True,
-        "modelType": "CLOUD",
-        "budgetMilliNodeHours": 8000,
-        "disableEarlyStopping": False,
-    }
-    training_task_inputs = json_format.ParseDict(training_task_inputs_dict, Value())
+
+    icn_training_inputs = definition.AutoMlImageClassificationInputs(
+        multi_label=True,
+        model_type=ModelType.CLOUD,
+        budget_milli_node_hours=8000,
+        disable_early_stopping=False
+    )
 
     training_pipeline = {
         "display_name": display_name,
         "training_task_definition": "gs://google-cloud-aiplatform/schema/trainingjob/definition/automl_image_classification_1.0.0.yaml",
-        "training_task_inputs": training_task_inputs,
+        "training_task_inputs": icn_training_inputs.to_value(),
```

Comment:
I rather prefer not to have additional method calls here.

Reply:
also note that this is simply a style issue with some hidden implication on sample generation.  please feel free to leave it as is for sample review.

Reply:
Changed.